pegasus/sites.v3/code.org/public/minecraft.haml

```diff
@@ -27,6 +27,17 @@ style_min: true
 %div{style: "clear:both"}
 .mc-container
   .mc-tutorials
+    .tutorial-box{style: "margin-bottom: 14px; display: #{DCDO.get('hoc_launch', nil).try(:[], 'mc_landing_2017') ? 'block' : 'none'}"}
```

Comment:
This is fine, though it might be more readable to yank the Ruby into a separate line just above.